client/src/components/Panel/Panel.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { Panel } from "./Panel";
+
+export default {
+  title: "Panel",
+  component: Panel,
+};
+
+const Template = (args) => <Panel {...args} />;
+
+const children = <div>Children</div>;
+
+export const Basic = Template.bind({});
+Basic.args = {
+  allowOverflow: true,
+  title: "title",
+  otherHeaderContent: "",
+  children,
+  sources: "",
+  id: 1,
+  glossary_keys: {},
+  footnotes: [],
+  isOpen: true,
+  style: {},
```

Comment:
Lots of unused props, but that's fine for now. Maybe just leave a `// TODO ...` comment saying something about making use of representing more props later on

Reply:
Done!